src/main/java/com/rultor/agents/req/Decrypt.java

```diff
@@ -58,11 +59,28 @@
     private final transient Profile profile;
 
     /**
+     * Proxy usage command string.
+     */
+    private final transient String proxy;
+
+    /**
+     * Ctor.
+     * @param prof Profile
+     * @param host Proxy host
+     * @param port Proxy port
+     */
+    Decrypt(final Profile prof, final String host, final int port) {
+        this.profile = prof;
+        this.proxy = this.createProxyString(host, port);
```

Comment:
there should be only one "working" ctor, all others should call it, see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#constructors